src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/HostEnvironmentExtensions.cs

```diff
@@ -0,0 +1,20 @@
+#nullable enable
+using System;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+
+public static class HostEnvironmentExtensions
+{
+    public const string IntegrationTestEnvironment = "IntegrationTests";
+
+    public static bool IsIntegrationTests(this IHostEnvironment? hostEnvironment) => 
```

Comment:
Not a biggy, but would it be better to make this singular i.e. `IsIntegrationTest` and `IntegrationTest`?

Reply:
No probs, I've updated that now ta